Library/Homebrew/cmd/install.rb

```diff
@@ -69,6 +69,10 @@ def install
           raise "No devel block is defined for #{f.full_name}"
         end
 
+        # TODO
+        # - Fix warning message if we're trying to install to directory in the
+        # Cellar, but it's a symlink to another one made by migrator while
+        # migrating.
```

Comment:
I think we can leave it as it is or remove it at all. I remember I spent some time trying to use more specified warning message, but then I saw that there were some cases when the message wasn't clear either, so I decided to left it as it was and add this TODO.